packages/rocketchat-lib/server/methods/leaveRoom.js

```diff
@@ -27,7 +26,17 @@ Meteor.methods({
 				throw new Meteor.Error('error-you-are-last-owner', 'You are the last owner. Please set new owner before leaving the room.', { method: 'leaveRoom' });
 			}
 		}
-
+		if (room.automatic) {
+
+			RocketChat.leave_automatic_channel(room.name);
+			// delete room if its the last user
```

Comment:
Can you link it here? It should be done so that if called from the server-side that it allows it.